pandas/tests/test_groupby.py

```diff
@@ -4594,9 +4594,6 @@ def test_frame_groupby_plot_boxplot(self):
         _skip_if_mpl_not_installed()
 
         import matplotlib.pyplot as plt
-        import matplotlib as mpl
```

Comment:
Yes it did, thus removed. Do you think the reason why these are added originally?
```
UserWarning: This call to matplotlib.use() has no effect because the the backend has already been chosen;
matplotlib.use() must be called *before* pylab, matplotlib.pyplot, or matplotlib.backends is 
```